components/auth/LoginView.tsx

```diff
@@ -1,6 +1,6 @@
 import { FC, useEffect, useState, useCallback } from 'react'
 import { Logo, Button, Input } from '@components/ui'
-import useLogin from '@framework/auth/use-login'
+import useLogin from '@commerce/auth/use-login'
```

Comment:
No code should be added outside the provider, if something is missing on the core we can discuss it in an external PR. Also `@framework` reffers to the hook from the provider, where you can extend the one from `@commerce`

Reply:
Okay, we'll submit a separate PR for the changes outside the provider, discuss the changes, and remove the changes from this PR.